### ISSUE TITLE
Fig Scripts insertion fails on `bash` 3.x.x 

### ISSUE THREAD

**Issue by graygt** [withfig/fig#2196]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 13.0.1 (22A400)

### Expected behaviour

MacBook-Pro-Sergey:~ gray$ fig run 
... entering port using fig interface=1338
... fig should execute this:
chrome-socks --port 1338

### Actual behaviour

MacBook-Pro-Sergey: fig run chrome-socks
... entering port using fig interface=1338
... fig executes this
MacBook-Pro-Sergey:~ gray$ 00~fig run chrome-socks --port 133801~
bash: 00~fig: command not found

There are 00~ before actual command and 01~ after it that corrupts execution.

### Environment

```yaml
fig-details:
  - 2.10.1
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Apple M1
  - cores: 8
  - mem: 16.00 GB
os-info:
  - macOS 13.0.1 (22A400)
environment:
  - shell: /bin/bash
  - terminal: iterm
  - cwd: /Users/gray
  - exe-path: /Users/gray/.fig/bin/fig
  - install-method: unknown
  - env-vars:
    - FIG_PID: 16468
    - TERM: xterm-256color
    - SHELL: /bin/bash
    - FIGTERM_SESSION_ID: 0fa7035b-6759-4492-b4dd-26378673c6db
    - FIG_SET_PARENT_CHECK: 1
    - PATH: /Users/gray/opt/miniconda3/condabin:/opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/Library/Apple/usr/bin:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/Users/gray/.fig/bin:/Users/gray/.local/bin:/Users/gray/Android/sdk/tools/bin:/Users/gray/Android/sdk/platform-tools:/Users/gray/Android/sdk/emulator
    - FIG_HOSTNAME: MacBook-Pro-Sergey.local
    - __CFBundleIdentifier: com.googlecode.iterm2
    - DISPLAY: /private/tmp/com.apple.launchd.VH8o61HB2M/org.xquartz:0
    - FIG_TERM: 2.10.1
```

**Comment by graygt** [comment]
@mschrage thanks! It works.
But I really can't find any reason why do people need to update default bash on their Macs, even if they quite advanced users that need software like fig. ;) So, there should be a lot users that encountered the same problem like me.

**Comment by grant0417** [comment]
This should be resolved now on 2.12.0 which is live
